skonfig/arguments.py

```diff
@@ -43,6 +43,12 @@ def get():
         metavar="path",
         help="initial manifest or '-' to read from stdin",
     )
+    parser.add_argument(
```

Comment:
Shouldn't this argument be `nargs='?'`?

If I run `skonfig -j example.com` I get the help.

Reply:
Nah, see comment above.